doc/howto/benchmarking-time.md

```diff
@@ -0,0 +1,45 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Display cell runtime in JupyterLab
+    keywords: jupyter, jupyterlab, sql
+    property=og:locale: en_US
+---
+
+# Benchmarking run-time in JupyterLab
+To record the time taken to run each cell
+in JupyterLab, we suggest using `jupyterlab-execute-time` 
+
+# Install jupyterlab-execute-time
+
+```sh
+pip install jupyterlab_execute_time
+```
+
+# Enable Record metadata Settings
```

Comment:
I think three headings for these is too much, you could structure like this

```
## Installation

## Usage

### (Optional) Store notebook metadata

### Sample screenshots

put your images here with some descriptive text



```

Reply:
Got it, will make the changes